cpp/src/community/ktruss.cu

```diff
@@ -23,6 +23,7 @@
 
 #include <cugraph/utilities/error.hpp>
 
+#include <rmm/thrust_rmm_allocator.h>
```

Comment:
Why would this be needed?  Seems like we're trying to get rid of these.

Reply:
Unfortunately, I could not find an other solution. Indeed, the legacy execution policy creation function is required by the Hornet code and I do not have access to it.